src/assets/i18n/fr.json

```diff
@@ -71,7 +71,7 @@
         }
       },
       "ui": {
-        "newFilter": "New filer"
+        "newFilter": "New filter"
```

Comment:
```suggestion
        "newFilter": "Nouveau filtre"
```

Reply:
The french version will come in a few weeks, for now the translation file is the same as english. Also, the french language selector is disabled for now

You don't need to fix this, but if you want to translate the whole file :D (i'll do it soon)